gcloud/bigquery/job.py

```diff
@@ -1140,3 +1153,307 @@ def _build_resource(self):
         self._populate_config_resource(configuration)
 
         return resource
+
+
+class _QueryConfiguration(object):
+    """User-settable configuration options for query jobs."""
+    # None -> use server default.
+    _allow_large_results = None
+    _create_disposition = None
+    _default_dataset = None
+    _destination_table = None
+    _flatten_results = None
+    _priority = None
+    _use_query_cache = None
+    _write_disposition = None
+
+
+class RunQueryJob(_BaseJob):
+    """Asynchronous job: query BQ tables.
```

Comment:
This makes me think the `BG` above is just meant to be `BQ`. Please just write out the whole word / phrase.